var/spack/repos/builtin/packages/py-biom-format/package.py

```diff
@@ -37,7 +37,7 @@ class PyBiomFormat(PythonPackage):
 
     variant('h5py', default=True, description='For use with BIOM 2.0+ files')
 
-    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools', type=('build', 'run'))
```

Comment:
I'm not seeing any evidence that this is a run-time dependency, are you?

Reply:
after you build py-biom-format, bin/biom contains this line:
```
from pkg_resources import load_entry_point
```